docs/section6/Rule6-10.md

```diff
@@ -0,0 +1,65 @@
+
+# Lighting - Rule 6-10  
+
+**Rule ID:** 6-10  
+**Rule Description:** Baseline building is modeled with occupancy sensor controls in applicable space types, i.e. employee lunch and break rooms, conference/meeting rooms, and classrooms (not including shop classrooms, laboratory classrooms, and preschool through 12th-grade classrooms).  
+**Appendix G Section Reference:** Section G3.1-6 Modeling Requirements for the baseline building  
+
```

Comment:
I think we can get rid of this rule because it's redundant to 6-13. 6-13 pulls occupancy sensor reductions from Table G3.7, and in that table occupancy sensor reduction (the third column) is set to none for these spaces.

Reply:
@mkarpman i see your point, however, i strongly recommend keeping this as 2 separate rules mostly because it aligns with the way things are specific in App G. Also, the  outcome for 6-10 is specific for baseline and outcome for 6-13 would be specific to the proposed. I agree that it is redundant but it gives us alignment with the rules.

Reply:
as discussed, we will merge 6-10 and 6-13. It will verify that proposed models OC using G3.7. This will automatically check that the spaces required to ahve OC in the B-RMR have them (i.e. P RMR is same as B RMR for those space types) as G 3.7 is set to 'none' for those space types.